atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -146,6 +146,8 @@ private InetSocketAddress getAddressForHostThrowUnchecked(String host) {
 
     public InetSocketAddress getAddressForHost(String host) throws UnknownHostException {
         if (config.addressTranslation().containsKey(host)) {
+            log.info("Found address translation mapping for host {}. The InetSocketAddress this host maps to is {}.",
```

Comment:
I'm not sure this is useful - if the translation is required, there will be very clear signal that the translation isn't working, since every cassandra operation will fail, plus this is very high volume.